test/main.cpp

```diff
@@ -22,15 +22,15 @@
 
 using namespace std;
 
-#ifdef NGRAPH_DISTRIBUTED
+#ifdef NGRAPH_DISTRIBUTED_ENABLE
 #include "ngraph/distributed.hpp"
 #endif
 
 int main(int argc, char** argv)
 {
-#ifdef NGRAPH_DISTRIBUTED
-    ngraph::Distributed dist;
-#endif
+    // #ifdef NGRAPH_DISTRIBUTED_ENABLE
+    //     ngraph::Distributed dist;
+    // #endif
```

Comment:
@wenzhe-nrv  Does this change mean that we can update [this documentation](http://ngraph.nervanasys.com/docs/latest/howto/distribute-train.html) to say "The default for making an nGraph build with the `DNGRAPH_DISTRIBUTED_ENABLE=TRUE` compilation flag will use two or more devices."  ?  

Or something similar.

Reply:
@indie yes, the documentation  would have to change 
now the user enables the flags based on the library of choice 
i.e. 
* the use of `NGRAPH_DISTRIBUTED_OMPI_ENABLE` enables OpenMPI
* the use of `NGRAPH_DISTRIBUTED_MLSL_ENABLE`  enables MLSL.

Note: this choice applies to `CPU` and `Interpreter` backend only
 For all other backends we support only `OpenMPI` at present. 
We can't use both the libraries simultaneously at present. 
We recommend the use of `Intel  MLSL` if the user is using CPU only backend.